src/core/expr/py_sort.h

```diff
@@ -34,13 +35,14 @@ class osort : public oobj
     private:
       oobj cols_;
       std::vector<bool>* reverse_;
+      std::vector<NaPosition>* na_position_;
```

Comment:
Since it's stored as a naked pointer, don't forget to free it in `m__dealloc__()`